packages/volto/theme/themes/pastanaga/extras/toolbar.less

```diff
@@ -752,3 +752,10 @@ body:not(.has-sidebar):not(.has-sidebar-collapsed) {
   }
 }
 // End Orphaned CSS
+
+// Toolbar handler color in homepage
+.contenttype-plone-site {
+  #toolbar .toolbar .toolbar-handler button:before {
```

Comment:
```suggestion
  #toolbar .toolbar-handler button:before {
```
@sabrina-bongiovanni you can keep the selector as specific as it was 3 levels plus the body override, there is no reason to have .toolbar in there if we look at how the background color is defined on the button
![teal-override](https://github.com/plone/volto/assets/152852/cbe65372-1a54-475d-b08a-b83f4bd4e6c4)

Personally, it bothers me that we have a generic button selector which is less efficient than a class selector.
I would add ".toolbar-handler-button" here https://github.com/plone/volto/blob/main/packages/volto/src/components/manage/Toolbar/Toolbar.jsx#L627
so that we always have such a class and use it instead of the simple button.
For published or draft it can still be
 #toolbar .toolbar-handler .published:before
#toolbar .toolbar-handler .private:before
but toolbar.less contains a bunch of button selectors which is a pity :)

Reply:
@ichim-david That makes sense. Changed the class and the selectors inside the .less file. Let me know if this is okay now :)

Reply:
> Personally, it bothers me that we have a generic button selector which is less efficient than a class selector. I would add ".toolbar-handler-button" here https://github.com/plone/volto/blob/main/packages/volto/src/components/manage/Toolbar/Toolbar.jsx#L627 so that we always have such a class and use it instead of the simple button. For published or draft it can still be #toolbar .toolbar-handler .published:before #toolbar .toolbar-handler .private:before but toolbar.less contains a bunch of button selectors which is a pity :)

@sabrina-bongiovanni I added this comment as my personal preference but I didn't make a suggestion for you to change this. I personally like this but technically speaking now since it modifies the testing structure it's a breaking change, although it simply adds a new class and not removing something.

The fix could have been fine with just this section that you've added https://github.com/plone/volto/pull/6126/files#diff-db29f0e3d44ba2993a71b0198daf2217864bf4318283240e32e3db9fcc25b32dR757

To be determined of @sneridagh opinion.